### PR TITLE
Nav bar henley

### PR DESCRIPTION
**Pull request by mik3ymikes**

### CODE REVIEW

File: src/app/nav-bar/nav-bar.component.html
Comment (mik3ymikes) [inline src/app/nav-bar/nav-bar.component.html:1]: practice comment

**Review by p134c0d3 — COMMENTED** [review]
JP - this looks good to me.

**Comment by Marcella44** [comment]
Looks good.

File: src/app/nav-bar/nav-bar.component.html
Comment (h-hovis) [inline src/app/nav-bar/nav-bar.component.html:1]: looks good

**Pull request merged**